src/components/AmplitudeAnalytics/index.ts

```diff
@@ -67,16 +67,16 @@ class UserModel {
     identify(mutate(new Identify()))
   }
 
-  set(key: string, value: string | number) {
+  set(key: string, value: string | number | boolean | string[] | number[]) {
     this.call((event) => event.set(key, value))
   }
 
-  setOnce(key: string, value: string | number) {
+  setOnce(key: string, value: string | number | boolean | string[] | number[]) {
     this.call((event) => event.setOnce(key, value))
   }
 
-  add(key: string, value: string | number) {
-    this.call((event) => event.add(key, typeof value === 'number' ? value : 0))
+  add(key: string, value: number) {
```

Comment:
should this also be `Value` or is it only a number?

Reply:
only number